### PR TITLE
Graph depth first

### PR DESCRIPTION
**Pull request by M-Ravenmoore**

### CODE REVIEW

**Comment by M-Ravenmoore** [comment]
[]Top-level README “Table of Contents” is updated
 []Feature tasks for this challenge are completed
 []Unit tests written and passing
 []“Happy Path” - Expected outcome
 []Expected failure
 []Edge Case (if applicable/obvious)
 []README for this challenge is complete
 []Summary, Description, Approach & Efficiency, Solution
 []Link to code
 []Picture of whiteboard

File: javascript/graphs/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+All appropriate neighbors can be retrieved from the graph
+Neighbors are returned with the weight between nodes included
+The proper size is returned, representing the number of nodes in the graph
+A graph with only one node and edge can be properly returned
+An empty graph properly returns null
+
+## API
+<!-- Description of each method publicly available in your Graph -->
+![BFS](./assets/breadthFirst-whiteboard.png)
+![get-edges](./assets/WB-CC37.jpg)
```
Comment (adrianhuebner) [inline javascript/graphs/README.md:25]: When you have a chance to come back to this add code and write some tests, then I will regrade for you!
